### ISSUE TITLE
 Google Cloud Healthcare API dataset picker dialog broken

### ISSUE THREAD

**Issue by JamesAPetts** [OHIF/Viewers#1449]
@dannyrb 

When using an oidc connection to a Google Cloud backend I get a bunch of messed up UI:

<img width="1392" alt="Screenshot 2020-02-17 at 17 54 00" src="https://user-images.githubusercontent.com/25818497/74676721-94c02080-51ae-11ea-95a1-e4dd027d665b.png">

Recent changes to the dialog modal probably broke this UI. All the data from GCP is being retrieved successfully, and I can access the data in the dialog if I mess around with the CSS in the browser.

**Comment by dannyrb** [comment]
When we created the `UIDialogService`, I believe we changed our Dialog implementation. We don't have Percy (visual tests) in place for Google HC functionality, so this slipped through.

We likely need to update the adapter to use the UI Dialog Service. When tinkering with James, we had to:

- Update the Dialog's position
- Update the Dialog's z-index

These would likely be addressed for use if the Dialog is opened inside the UI Dialog Container, which is what the UI Dialog Service does for us.

See: https://docs.ohif.org/services/ui/ui-dialog-service.html
